factorio-map-process.sh

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env bash
+
```

Comment:
As I think I mentioned on some other line, I'd like to see
```suggestion
set -euo pipefail
```
as bash options on this script.

Some people like to have it as my suggestion on line 2, others like to have it as part of the shbang line, like this:
```bash
#!/usr/bin/env bash -euo pipefail
```
I prefer the first option.

Blog post about this:
https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/

The reasoning is that Shell scripting tends to act like a bunch of commands, not a programing language, but once you start to get long scripts like this, you tend to need programing language error handling features. Those 3 (error exit, error unset, error pipe fail) are good defaults.